Demo/UI Tests/Venmo UI Tests/Venmo_UITests.swift

```diff
@@ -18,26 +18,33 @@ class Venmo_UITests: XCTestCase {
         demoApp.launchArguments.append("-Integration:BraintreeDemoCustomVenmoButtonViewController")
         demoApp.launch()
 
-        waitForElementToBeHittable(demoApp.buttons["Venmo (custom button)"])
+        waitForElementToBeHittable(demoApp.buttons["Venmo (custom button)"], timeout: 15.0)
```

Comment:
Default `10.0` timeout period causes false negatives sometimes because the demo app fetches a client token before rendering buttons.

Reply:
Is it worth bumping all of the default timeouts to 15? I know these tests have the most flakiness overall so may be good to apply this across the board.

Reply:
That's a good call I'll make that the default. They tend to flake out when the HTTP call to fetch the client token hangs.